examples/dynamic_api/storageclass-wekafs-dir-api.yaml

```diff
@@ -9,8 +9,15 @@ allowVolumeExpansion: true
 parameters:
   volumeType: dir/v1
   filesystemName: default
-  # if the values below are not set, quota will not be enforced for this storage class
+  # optional parameters setting UID, GID and permissions on volume
+  # UID of the volume owner, default 0 (root)
+  ownerUid: "1000"
```

Comment:
maybe put it as a separate example or comment out?
As of right now, this basically makes the default example not usable

Reply:
Done